src/Cedar/BridgeWin32.c

```diff
@@ -1161,7 +1161,9 @@ void Win32EthMakeCombinedName(char *dst, UINT dst_size, char *nicname, char *gui
 
 	if (IsEmptyStr(guid) == false)
 	{
-		Format(dst, dst_size, "%s (ID=%010u)", nicname, Win32EthGenIdFromGuid(guid));
+		// Allow to combine "FriendlyName" consisting of a NULL character and ID.
+		//Format(dst, dst_size, "%s (ID=%010u)", nicname, Win32EthGenIdFromGuid(guid));

```

Comment:
What is the reason for leaving old code in comments?  The old code is kept in git history and I don't see any good reason to do such thing everywhere.

Reply:
personally I prefer to remove comments.
also `//` is C++ standard while source is C


but honestly, I think it is cosmetic (and we can cut comments in follow up PR for example :)

Reply:
>everywhere ?
It is false.
Are you a board member of this repository ?

Reply:
I don't get the point of the question.

Reply:
BOARD MEMBERS OF THIS REPOSITORY is described here

https://github.com/SoftEtherVPN/SoftEtherVPN?tab=readme-ov-file#board-members-of-this-repository


Your name not found here. So, I guess that you are not a board member of this repository.

You should say your preference to the BOARD MEMBERS OF THIS REPOSITORY not me.

Reply:
@hiura2023 , it is easy to find out the list of board members, why are you asking @metalefty whether he is a board member or not ?